### ISSUE TITLE
Add check for valid range of telescope IDs in names:validateTelescopeIDName()

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/gammasim-tools#225]
Requires knowledge about the telescopes for a given site.

**Comment by GernotMaier** [comment]
After looking into it, I suggest to close this issue and remove the FIXME in that method ([here](https://github.com/gammasim/gammasim-tools/blob/da0fd3035c7eb16bd2ac9db6ebe19b3040f13710/simtools/util/names.py#L85)).

`names.py` is about name validation, not about validating the values. This should happen somewhere else.

Ok with closing it?

**Comment by orelgueta** [comment]
Yes, agree. Having this validation would also create a lot of work later when we start adding real telescopes. It's much better to simply let the DB query fail in case a wrong telescope is given. Opened #402 to close this issue.
